src/Avalonia.DesignerSupport/Remote/HtmlTransport/HtmlTransport.cs

```diff
@@ -262,5 +260,75 @@ private void OnSignalTransportException(IAvaloniaRemoteTransportConnection arg1,
             _onException?.Invoke(this, ex);
         }
         #endregion
+
+        private static object ParseMessage(string message)
+        {
+            var parts = message.Split(':');
+            var key = parts[0];
+            if (key.Equals("frame-received", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Wouldn't it be better to use `OrdinalIgnoreCase` here?

Reply:
Yes, I don't think we need invariant culture to compare these strings.